changelog.d/20221102_223217_sirosen_timer_prompt.md

```diff
@@ -0,0 +1,9 @@
+### Enhancements
+
+* Add `globus timer create transfer` as a new command for creating new timers
+
+** The command prompts for login if data_access consents are detected as a
+   requirement
+
+* `globus login` will now request a `view_consents` scope and new logins may
+  be required in order to run some commands
```

Comment:
```suggestion

* `globus session consent` now supports a `--timer-data-access` flag, specifically
   to help support timer creation
```

I removed the new scope (not actually needed), and added a flag. I think this reflects the changes pretty well.